### PR TITLE
Vertigo seismic strike power

### PR DESCRIPTION
**Pull request by CDVoidwalker**
I guess it needs also your refactor of targettypes @jrb0001

### CODE REVIEW

**Review by jrb0001 — CHANGES_REQUESTED** [review]
It doesn't depend on the refactoring of the targettype as far as I can see.

Just the tooltip on the Pyramid is missing, everything else looks fine.

File: rules/nod-aircraft.yaml [hunk 1]
```diff
@@ -556,6 +556,15 @@ VERTIGO2:
 		Image: vertigo
 	RenderSprites:
 		Image: vertigo
+
```
Comment (GraionDilach) [inline rules/nod-aircraft.yaml]: Superflous tabs.

File: weapons/nod-weapons.yaml [hunk 2]
```diff
@@ -938,7 +938,98 @@ VertigoBombNapalm:
 		Weapon: NapalmShrapnel
 		Amount: 30
 		AimChance: 0
-
+
```
Comment (GraionDilach) [inline weapons/nod-weapons.yaml]: Here too.

**Review by jrb0001 — APPROVED** [review]
Looks good to me.

**Pull request merged**